lib/parser.rb

```diff
@@ -0,0 +1,15 @@
+class Parser
+    attr_accessor :file_pathname
+
+    def initialize(file_pathname)
+        @file_pathname = file_pathname
+    end
+
+    def get_first_line
+        if(File.file?(@file_pathname))
```

Comment:
Em ruby nao é necessario utilizar parenteses, entao vc pode fazer assim:

```suggestion
        if File.file?(@file_pathname)
```